spec/requests/image_uploads_spec.rb

```diff
@@ -75,8 +75,8 @@
     context "when uploading rate limiting works" do
       before do
         sign_in user
-        allow(Rails).to receive(:cache).and_return(memory_store)
-        Rails.cache.clear
+        RedisRailsCache = ActiveSupport::Cache::MemoryStore.new
```

Comment:
We should add an `after`  block restoring `RedisRailsCache` to `ActiveSupport::Cache::NullStore.new` which is the default

Reply:
@rhymes did you mean this 

```
  RedisRailsCache = ActiveSupport::Cache::MemoryStore.new
RedisRailsCache.clear
  RedisRailsCache = ActiveSupport::Cache::NullStore.new
```
?

Reply:
```ruby
before do
  RedisRailsCache = ActiveSupport::Cache::MemoryStore.new
end

after do
  RedisRailsCache = ActiveSupport::Cache::NullStore.new
end
```

something like this. The objective is to restore the resources to the value they had before when the test block exits. This is very important for the cache because we don't want to activate the cache for all tests in the suite